tests/unittests/TraceEventsTest.cpp

```diff
@@ -314,6 +360,21 @@ TEST_P(TraceEventsTest, manualAndAutomatic) {
   cctx.backendOpts.autoInstrument = true;
   EE_.compile(cctx);
 
+  std::vector<std::pair<std::string, std::string>> expectedKinds;
```

Comment:
can you make this as a helper method which extracts node kinds from the given function and reuse in both tests

Reply:
Ok